assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -70,6 +71,13 @@ beforeEach(() => {
 
 jest.mock("../../../src/api")
 
+jest.mock("../../../src/hooks/useNearestIntersection", () => ({
+  __esModule: true,
+  useNearestIntersection: jest.fn(() => {
+    return { is_loading: true }
+  }),
+}))
```

Comment:
suggestion: I believe this should work without the extra options?(at least if combined with adding the mock to `beforeEach`)
(yet another piece of tech debt that makes it hard to contribute due to how often it's used)

```suggestion
jest.mock("../../../src/hooks/useNearestIntersection")
```

Reply:
The api mock was maybe accidentally removed and is still needed, but otherwise yes! It does still work without all the extra options, thank you

Reply:
> The api mock was maybe accidentally removed and is still needed
Whoops yeah didn't mean to catch `api` in this suggestion